fxa-oauth-server/lib/config.js

```diff
@@ -66,6 +66,12 @@ const conf = convict({
       env: 'CLIENT_MANAGEMENT_ENABLED'
     }
   },
+  clientIdToServiceNames: {
```

Comment:
Note, I add this instead of including this straight into `clients` because [cloudops-deployment has this file that can be included into config](https://github.com/mozilla-services/cloudops-deployment/blob/master/projects/fxa/puppet/yaml/app/fxa.prod.yaml#L1) across multiple repos.

Reply:
I hope this can be refactored out to use https://github.com/mozilla/fxa-auth-server/blob/2e1b01f87b102eb635d4604d199ed35d298194f9/lib/oauthdb/index.js#L68 some day ⌛️

Reply:
Would be good. 👍